include/rive/artboard.hpp

```diff
@@ -161,6 +164,7 @@ namespace rive {
 
         std::unique_ptr<StateMachineInstance> stateMachineAt(size_t index);
         std::unique_ptr<StateMachineInstance> stateMachineNamed(const std::string& name);
+        std::unique_ptr<StateMachineInstance> defaultStateMachineInstance();
```

Comment:
We could name this ```defaultStateMachine``` as an overload of the one in Artboard, but it may cause confusion.